### ISSUE TITLE
ZipTest fails and aff4-test crashes 

### ISSUE THREAD

**Issue by uckelman-sf** [Velocidex/c-aff4#68]
```
[juckelman@midas tests]$ ./aff4-test 
[==========] Running 21 tests from 10 test cases.
[----------] Global test environment set-up.
[----------] 3 tests from MemoryDataStoreTest
[ RUN      ] MemoryDataStoreTest.IncompatibleGet
[       OK ] MemoryDataStoreTest.IncompatibleGet (4 ms)
[ RUN      ] MemoryDataStoreTest.StorageTest
[       OK ] MemoryDataStoreTest.StorageTest (3 ms)
[ RUN      ] MemoryDataStoreTest.TurtleSerializationTest
[       OK ] MemoryDataStoreTest.TurtleSerializationTest (7 ms)
[----------] 3 tests from MemoryDataStoreTest (14 ms total)

[----------] 1 test from AFF4ObjectCacheTest
[ RUN      ] AFF4ObjectCacheTest.TestLRU
[       OK ] AFF4ObjectCacheTest.TestLRU (3 ms)
[----------] 1 test from AFF4ObjectCacheTest (3 ms total)

[----------] 1 test from StreamTest
[ RUN      ] StreamTest.StringIOTest
[       OK ] StreamTest.StringIOTest (0 ms)
[----------] 1 test from StreamTest (0 ms total)

[----------] 1 test from FileBackedStreamTest
[ RUN      ] FileBackedStreamTest.FileBackedObjectIOTest
[       OK ] FileBackedStreamTest.FileBackedObjectIOTest (3 ms)
[----------] 1 test from FileBackedStreamTest (3 ms total)

[----------] 4 tests from ZipTest
[ RUN      ] ZipTest.CreateMember
zip_test.cc:46: Failure
Value of: file.get()
  Actual: false
Expected: true
Unable to create zip file
[  FAILED  ] ZipTest.CreateMember (2 ms)
[ RUN      ] ZipTest.OpenMemberByURN
zip_test.cc:46: Failure
Value of: file.get()
  Actual: false
Expected: true
Unable to create zip file
[  FAILED  ] ZipTest.OpenMemberByURN (3 ms)
[ RUN      ] ZipTest.ConcatenatedVolumes
zip_test.cc:46: Failure
Value of: file.get()
  Actual: false
Expected: true
Unable to create zip file
[  FAILED  ] ZipTest.ConcatenatedVolumes (3 ms)
[ RUN      ] ZipTest.testStreamedSegment
zip_test.cc:46: Failure
Value of: file.get()
  Actual: false
Expected: true
Unable to create zip file
[  FAILED  ] ZipTest.testStreamedSegment (3 ms)
[----------] 4 tests from ZipTest (11 ms total)

[----------] 2 tests from AFF4ImageTest
[ RUN      ] AFF4ImageTest.OpenImageByURN
[2019-01-23 17:29:18.562] [aff4] [error] Unable to open backing URN /tmp/aff4_test.zip
[2019-01-23 17:29:18.563] [aff4] [error] Unable to open backing URN /tmp/aff4_test.zip
[2019-01-23 17:29:18.563] [aff4] [error] Unable to open backing URN /tmp/aff4_test.zip
terminate called after throwing an instance of 'std::future_error'
  what():  std::future_error: No associated state
Aborted (core dumped)
```

**Comment by uckelman-sf** [comment]
This is 5fe739cc, after commenting out `rdf_tests.cc` from `Makefile.am` in order to work around the compilation problem in Issue #67.

**Comment by uckelman-sf** [comment]
Working to fix this on fix-issue-68 branch.

**Comment by uckelman-sf** [comment]
The problems noted here are fixed in PR #70. Tests may now be run using `make check`.
